app/assets/javascripts/activities.es6

```diff
@@ -0,0 +1,66 @@
+/* global TapBase, Pusher */
+{
+  'use strict';
+
+  class Activities extends TapBase {
+    start(el) {
+      super.start(el);
+      this.setupPusher();
+      this.groupAllActivities();
+    }
+
+    setupPusher() {
+      const channel = Pusher.instance.subscribe('telephone_appointment_planner');
+      channel.bind(this.config.event, this.handlePushEvent.bind(this));
+    }
+
+    handlePushEvent(payload) {
+      let $activity = $(payload.body);
+      $activity
+        .hide()
+        .prependTo(this.getActivityGroup($activity))
+        .addClass('t-dynamically-loaded-activity')
+        .fadeIn();
+    }
+
+    groupAllActivities() {
+      if (!this.groups) { this.groups = {}; }
+
+      this.$el.hide();
+      this.$el.removeClass('hide');
+      let $activities = this.$el.find('.js-activity');
```

Comment:
Speaking of which, should we rely on just the class rather than using jQuery's `hide()` method as well? Could get confusing having both – jQuery just adds inline styles which will trump our CSS.